.github/workflows/test.yaml

```diff
@@ -7,7 +7,7 @@ on:
     - cron:  '0 0 * * 0'
   pull_request_target:
```

Comment:
We'll need to change this to just `pull_request:` since this workflow is no longer accessing any secrets. I opened an issue to elaborate https://github.com/uwhackweek/jupyterbook-template/issues/49